modules/oadp-creating-restore-cr.adoc

```diff
@@ -55,3 +55,15 @@ $ oc get restore -n openshift-adp <restore> -o jsonpath='{.status.phase}'
 $ oc get all -n <namespace> <1>
 ----
 <1> Namespace that you backed up.
+
+. If you use Restic to restore DeploymentConfigs or if you use post-restore hooks, run a cleanup script by entering the following command:
++
+[source,terminal]
+----
+$ oc dc-restic-post-restore.sh <restore-name>
```

Comment:
I believe it should be:
**bash** dc-restic-post-restore.sh \<restore-name\>

@kaovilai/ @sseago correct my if I'm wrong

Reply:
@mperetzred That's right. This is a bash script, not an oc subcommand

Reply:
Make sure to point user that this script can be obtained per info at
https://github.com/openshift/openshift-docs/blame/e93998f8b4102e76bf0dc9c35be6dbf388bd5e91/backup_and_restore/application_backup_and_restore/backing_up_and_restoring/restoring-applications.adoc#L14-L17

Reply:
Corrected